auth/http.go

```diff
@@ -1098,6 +1117,23 @@ func httpHandlerSetGroupRoles(central *ImqsCentral, w http.ResponseWriter, r *ht
 	broadcastGroupChange(central, r, groupname)
 }
 
+func Diff(a authaus.PermissionList, b authaus.PermissionList) authaus.PermissionList {
```

Comment:
Rather make the `Diff` a function on `authaus.PermissionList`, incase we one day want to create another Diff function (silly go an its lack of method overloading)

All thou this means that you would have to make changes in authaus I believe

Reply:
ok will do